src/common/bfloat16.h

```diff
@@ -78,7 +78,9 @@ class bfloat16_t {
     }
 
     static void cvt_float_to_bfloat16(const float *src, bfloat16_t *dst, int size);
+    static void batch_cvt_float_to_bfloat16(const float *src, bfloat16_t *dst, int size);
```

Comment:
@changqi1 Do you accept adding it in bfloat16.h?

Reply:
@pujiang2018  @marvin-Yu Sure. It is great to add batch cvt. But the number 1024 is not make sense.